docker/images/pinot/etc/jmx_prometheus_javaagent/configs/broker.yml

```diff
@@ -59,6 +59,12 @@ rules:
   labels:
     database: "$2"
     table: "$1$3"
+- pattern: "\"org\\.apache\\.pinot\\.common\\.metrics\"<type=\"BrokerMetrics\", name=\"pinot\\.broker\\.requestSize\\.(([^.]+)\\.)?([^.]*)\"><>(\\w+)"
+  name: "pinot_broker_requestSize_$4"
+  cache: true
+  labels:
+    database: "$2"
+    table: "$1$3"
```

Comment:
Do we want to delimit `$1` from `$3`?

Reply:
$1 matches the whole `databaseName.` part hence no need to delimit explicitly.